exporter/collector/config.go

```diff
@@ -140,6 +142,18 @@ type MetricConfig struct {
 	EnableSumOfSquaredDeviation bool `mapstructure:"sum_of_squared_deviation"`
 }
 
+// WALConfig defines settings for the write ahead log. WAL buffering writes data
+// points in-order to disk before reading and exporting them. This allows for
```

Comment:
Is there a way to do in-memory only WAL?

Reply:
Not with this library: https://github.com/tidwall/wal/blob/e70ddec4a12f90db734d4d90df98b5088bd6d417/wal.go#L159

Reply:
I think handling network outages is the priority.  5-min outage with even a potential collector crash/reboot would be handled by the on-disk portion here.

In-memory is a nice to have, unless the WAL is not viable for some reason.